### PR TITLE
Create search page

### PR DESCRIPTION
**Pull request by AndrewKchn**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,11 @@
             <artifactId>selenium-java</artifactId>
             <version>3.141.59</version>
         </dependency>
+        <dependency>
```
Comment (YuraBra) [inline pom.xml:21]:
```
<dependency>
            <groupId>org.seleniumhq.selenium</groupId>
            <artifactId>selenium-java</artifactId>
            <version>3.141.59</version>
        </dependency>
```

**// are there any differences? Won't conflict?**

File: src/main/java/core/BasePage.java [hunk 2]
```diff
@@ -26,10 +26,12 @@
 
 
     protected WebDriver driver;
+    protected String url = Config.site;
```
Comment (YuraBra) [inline src/main/java/core/BasePage.java]: **explain please**

File: src/main/java/page/SearchPage.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    public SearchPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void fillSearchField(String desireItem){
+        fillField(searchField,desireItem);
+    }
+
+    public void sellectKategory(String categotiOfItem){
```
Comment (YuraBra) [inline src/main/java/page/SearchPage.java]: **sellectKategory - misprints**

Comment (YuraBra) [inline src/main/java/page/SearchPage.java]: **categotiOfItem**

File: src/main/java/page/SearchPage.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        chooseDataFromDropDownMenu(category,categotiOfItem);
+    }
+
+    public void selectSearchInSubkategories(){
```
Comment (YuraBra) [inline src/main/java/page/SearchPage.java]: **selectSearchInSubkategories**

File: src/main/java/page/SearchPage.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    public SearchPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public void fillSearchField(String desireItem){
+        fillField(searchField,desireItem);
+    }
+
+    public void sellectKategory(String categotiOfItem){
+        chooseDataFromDropDownMenu(category,categotiOfItem);
```
Comment (YuraBra) [inline src/main/java/page/SearchPage.java:39]: **categotiOfItem**

**Review by NataliiaKriuchkova — APPROVED** [review]

**Pull request closed without merging**